roles/load-ycsb/templates/configmap.yml.j2

```diff
@@ -0,0 +1,245 @@
+    #
+    # Unless required by applicable law or agreed to in writing, software
+    # distributed under the License is distributed on an "AS IS" BASIS,
+    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+    # implied. See the License for the specific language governing
+    # permissions and limitations under the License. See accompanying
+    # LICENSE file.
+    # Yahoo! Cloud System Benchmark
+    # Workload E: Short ranges
+    #   Application example: threaded conversations, where each scan is for the posts in a given thread (assumed to be clustered by thread id)
+    #
+    #   Scan/insert ratio: 95/5
+    #   Default data size: 1 KB records (10 fields, 100 bytes each, plus key)
+    #   Request distribution: zipfian
+    # The insert order is hashed, not ordered. Although the scans are ordered, it does not necessarily
+    # follow that the data is inserted in order. For example, posts for thread 342 may not be inserted contiguously, but
+    # instead interspersed with posts from lots of other threads. The way the YCSB client works is that it will pick a start
+    # key, and then request a number of records; this works fine even for hashed insertion.
+    recordcount={{ycsb.recordcount}}
+    operationcount={{ycsb.operationcount}}
+    workload=com.yahoo.ycsb.workloads.CoreWorkload
+    readallfields=true
+    readproportion=0
+    updateproportion=0
+    scanproportion=0.95
+    insertproportion=0.05
+    requestdistribution=zipfian
+    maxscanlength=100
+    scanlengthdistribution=uniform
+  workloadf : |
+    # Copyright (c) 2010 Yahoo! Inc. All rights reserved.
+    #
+    # Licensed under the Apache License, Version 2.0 (the "License"); you
+    # may not use this file except in compliance with the License. You
+    # may obtain a copy of the License at
+    #
+    # http://www.apache.org/licenses/LICENSE-2.0
+    #
+    # Unless required by applicable law or agreed to in writing, software
+    # distributed under the License is distributed on an "AS IS" BASIS,
+    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+    # implied. See the License for the specific language governing
+    # permissions and limitations under the License. See accompanying
+    # LICENSE file.
+    # Yahoo! Cloud System Benchmark
+    # Workload F: Read-modify-write workload
+    #   Application example: user database, where user records are read and modified by the user or to record user activity.
+    #
+    #   Read/read-modify-write ratio: 50/50
+    #   Default data size: 1 KB records (10 fields, 100 bytes each, plus key)
+    #   Request distribution: zipfian
+    recordcount={{ycsb.recordcount}}
+    operationcount={{ycsb.operationcount}}
+    workload=com.yahoo.ycsb.workloads.CoreWorkload
+    readallfields=true
+    readproportion=0.5
+    updateproportion=0
+    scanproportion=0
+    insertproportion=0
+    readmodifywriteproportion=0.5
+    requestdistribution=zipfian
```

Comment:
You might take a look at the recent work I've done for fio that allows for overriding config defaults via the CR file. Something similar here might provide even more flexibility for the power user.

Reply:
Correct, I just got a email from Elko and he mentioned that we should look at changing this to uniform. I should template this out too...